### ISSUE TITLE
Better blurred<->unblurred transition

### ISSUE THREAD

**Issue by alfredjophy** [WillPower3309/swayblur#12]
An enhancement suggestion

Right now the delay just pauses before switching the wallpaper which is too sudden

If a series of images with a blur gradually increasing/decreasing is generated based on the delay, then that can be merged into a GIF and played by oguri...

This would make the transition from blurred to unblurred way smoother when you close a window

**Comment by WillPower3309** [comment]
Have you tried playing around with the `-a` argument?

**Comment by alfredjophy** [comment]
yes, initially the slow animations is smoother but the switching from blurred to unblurred is too quick
